src/libs/Notification/PushNotification/ForegroundNotifications/index.ios.ts

```diff
@@ -20,7 +21,9 @@ function disableForegroundNotifications() {
     Airship.push.iOS.setForegroundPresentationOptionsCallback(() => Promise.resolve([]));
 }
 
-export default {
+const ForegroundNotifications: ForegroundNotificationsType = {
```

Comment:
I'm not sure this is the best name for these functions. Perhaps `ForegroundNotificationSetup`, or `ForegroundNotificationLifecycleFuncs` is more descriptive?

Reply:
Usually we create the lib type with the same as the folder, but in this case I had to use `ForegroundNotificationsType` to differentiate with the variable name. Instead, we could use `ForegroundNotificationsModule` like we did [here](https://github.com/Expensify/App/blob/main/src/libs/ControlSelection/types.ts#L3). WDYT @Julesssss ?

Reply:
Yeah that sounds good!